docs/sphinx_source/c_functions.rst

```diff
@@ -20,4 +20,4 @@ A generic new skeleton for a function looks like this:
            const float value_1) {
         auto df1 = df.Define(....);
         return df1;
-    }
\ No newline at end of file
+    }
```

Comment:
I can't see here on github what the actual diff is. Unless it's really an intended change, better remove it to keep commits clean.